### PR TITLE
[runc] Initial integration

### PR DESCRIPTION
**Pull request by AdamKorcz**

### CODE REVIEW

**Comment by AdamKorcz** [comment]
@cyphar we have also set up initial integration of runc here. Who can we add as maintainers for this one?

**Comment by cyphar** [comment]
Thanks, and as with umoci please use the list [from the runc project's MAINTAINERS file](https://github.com/opencontainers/runc/blob/master/MAINTAINERS).

**Comment by AdamKorcz** [comment]
@jonathanmetzman @inferno-chromium Ready for review.

File: projects/runc/configs_fuzzer.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package configs
```
Comment (jonathanmetzman) [inline projects/runc/configs_fuzzer.go:1]: File needs a license.

File: projects/runc/id_map_fuzzer.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package system
```
Comment (jonathanmetzman) [inline projects/runc/id_map_fuzzer.go:1]: File needs a license.

File: projects/runc/user_fuzzer.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
+package user
+
```
Comment (jonathanmetzman) [inline projects/runc/user_fuzzer.go:2]: File needs a license.

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**